packages/aft/lib/src/commands/generate/generate_workflows_command.dart

```diff
@@ -335,6 +354,20 @@ jobs:
       package-name: $packageNameToTest
 ''';
 
+    if (setExitIfChanged) {
+      exitIfChanged(iosWorkflowFile, iosWorkflowContents);
+    }
+
     iosWorkflowFile.writeAsStringSync(iosWorkflowContents);
   }
+
+  void exitIfChanged(File workflowFile, String content) {
+    final currentContent = workflowFile.readAsStringSync();
+    if (currentContent != content) {
+      logger
+        ..error('Workflows are not up to date.')
+        ..error('Run `aft generate workflows` to regenerate them.');
+      exit(1);
+    }
+  }
```

Comment:
Can maybe consolidate this logic a bit more

```suggestion
  void writeWorkflowFile(File workflowFile, String content) {
    if (!workflowFile.existsSync()) {
      workflowFile.createSync();
    }
    final currentContent = workflowFile.readAsStringSync();
    if (currentContent != content && setExitIfChanged) {
      logger
        ..error('Workflows are not up to date.')
        ..error('Run `aft generate workflows` to regenerate them.');
      exit(1);
    }
    workflowFile.writeAsStringSync(content);
  }
```

Reply:
Good call. Updated.